test/kubernetes/testutils/cluster/istio.go

```
@@ -15,9 +15,13 @@ import (
 	"github.com/solo-io/go-utils/contextutils"
 )
 
+const (
+	defaultIstioVersion = "1.19.9"
```

So the default was intentionally removed after this conversation: https://github.com/solo-io/gloo/pull/9458#discussion_r1594091933 I'm not sure we want a hardcoded default instead of failing loudly if it's not set in the env. Not having a default also makes it easier to have the Istio version set as an env to test across Istio versions and make sure we're not using a default in those tests.

---

added this because whenever running locally, we need to always add ISTIO_VERSION=... otherwise the tests will panic. i started updating all the readmes to add in the ISTIO_VERSION but realized it would be cleaner to just default it in one place (i.e. i agree with your reply in the linked comment above).

we already are defaulting it in some places e.g. setup-kind `ISTIO_VERSION="${ISTIO_VERSION:-1.19.9}"` so i don't think it's unreasonable to default it here in the tests too, to the same version

---

I think we do want it to loudly panic if the `ISTIO_VERSION` is not set in the env. 

The setup-kind script is run from ci versus this would be hard coded in the test code. This adds an additional thing to upgrade as we change supported Istio version and is more likely to get out of sync.

The follow up to the Istio tests is to add a test matrix across Istio versions. If one of those versions is not correctly set in the github workflow matrix, it will still pass the test because we have this default hard coded instead of failing loudly. 

GME also doesn't hard code the version in the test setup and only relies on the env: https://github.com/solo-io/gloo-mesh-enterprise/blob/main/test/scripts/setup-funcs.sh#L334

---

reverted 7d370f1